docs/release-notes.rst

```diff
@@ -6,6 +6,190 @@
  Release Notes
 ###############
 
+**************
+ Version 0.35
+**************
+
+Version 0.35.0
+==============
+
+**Release Date:** August 02, 2024
+
+**Breaking Changes**
+
+-  Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name``
+   for better clarity and to support multiple resource managers.
+
+   -  Resource managers operate relative to the specific cluster providing resources for Determined
+      tasks, so changing the cluster will affect the resource manager's responses.
+   -  The ``cluster_name`` must be unique for all resource managers when deploying multiple resource
+      managers in Determined.
+   -  During upgrade, replace ``name`` with ``cluster_name`` in the ``resource_manager`` section of
```

Comment:
When upgrading, specify a `resourceManager.clusterName` in your `values.yaml` to override `resource_manager.name` and/or remove the `name` field from your `resource_manager` config altogether.
 
- For additional resource managers, you must change `additional_resource_manager[i].name` to `additional_resource_manager[i].cluster_name` in your `values.yaml`.